addons/mail/static/src/utils/common/hooks.js

```diff
@@ -195,8 +195,8 @@ export function useHover(refNames, { onHover, onAway, stateObserver, onHovering
     }
 
     if (stateObserver) {
-        useEffect(() => {
-            if (lastHoveredTarget && !lastHoveredTarget.ref.el) {
+        useEffect((open) => {
+            if ((lastHoveredTarget && !lastHoveredTarget.ref.el) || !open) {
```

Comment:
Should put a comment that the `open` part is to work with `useDropdownState()` that notifies "close" transition but the ref is still there an animation longer.